### DISCUSSION TITLE
Parse server - so why?

### DISCUSSION THREAD

**Discussion by FELS-Zak** [pocketbase/pocketbase#103] [Q&A] [answered]
HI - I freakin love PB. but I've got to ask, why did it need to exist when we hvae things like PARSE server ? I know it's in GO but are there any scope improvements over it?

**Comment by ganigeorgiev** [comment, marked as answer]
I've created PocketBase because of my other open source project - [Presentator](https://github.com/presentator/presentator) (you can read more in [Presentator #183](https://github.com/presentator/presentator/issues/183)).

In short, I wanted something that is self-contained and doesn't require installation so that anyone can self-host the platform.

**Reply by FELS-Zak** [reply]
Thanks for creating it 👍

**Reply by FELS-Zak** [reply]
how long did it take?

**Reply by ganigeorgiev** [reply]
Oh, I don't know, it's hard to say. I've been working on PocketBase alongside my day-to-day job since mid-December 2021, so roughly compressed is ~3-5 months.

The last 3-weeks I've taken a longer break from my job so that I could release PocketBase and fix eventual initial issues. The next week I'll have to return to the office and probably things will slow down a little, but I'll try my best to continue supporting it (soon will release a roadmap for v1.0.0).

**Reply by FELS-Zak** [reply]
Thank you. - pretty impressed - would love to learn how you planned this out -
